src/Analyzer/CodeAnalyzer.cs

```diff
@@ -73,7 +73,24 @@ public CodeAnalyzer() : base(csharpExtension)
                 additionalFiles.Add(new AnalyzerAdditionalFile(sonarConfigurationPath));
             }
 
-            diagnosticsRunner = new DiagnosticsRunner(GetAnalyzers(), GetUtilityAnalyzers(), additionalFiles.ToArray());
+            UpdatePatternIdentifiers();
```

Comment:
Consider inlining the code of the helper function here. It's called only one and it's side-effectful.